### DISCUSSION TITLE
发送cq码表情是这样吗？发送没有报错，但是接受到消息呢？

### DISCUSSION THREAD

**Discussion by 867525168** [OrgEleCho/EleCho.GoCqHttpSdk#19] [Q&A] [answered]
> await session.SendGroupMessageAsync(groupId, new CqMessage(new CqFaceMsg(0)));

> if (CqFaceMsg.FromName(sendContent1) is CqFaceMsg face)
   await session.SendGroupMessageAsync(groupId, new CqMessage(face));

这样发送cq码表情是这样吗？发送没有报错，但是接受到消息呢？

**Comment by SlimeNull** [comment, marked as answer]
一个使用消息匹配拓展插件的示例

```csharp
[CqMessageMatch("^#face (?<content>.*)")]
public async Task Face(CqGroupMessagePostContext context, string content)
{
    if (CqFaceMsg.FromName(content) is CqFaceMsg face)
        await ActionSession.SendGroupMessageAsync(context.GroupId, new CqMessage(face));
}
```

经测试, 是可用的
